cmd/main.go

```diff
@@ -34,7 +38,26 @@ func init() {
 		"To provide feedback or request new features please open issues at https://github.com/docker/compose"
 }
 
-func main() {
+func standaloneMain() int {
+	dockerCli, err := command.NewDockerCli()
```

Comment:
```console
$ docker-compose --host tcp://localhost:2345 up
Error: unknown flag: --host
```
need to first [Convert](https://github.com/docker/compose-switch/blob/master/redirect/convert.go#L27) the command line to be cli-plugin compliant